hibernate-reactive-core/src/main/java/org/hibernate/reactive/context/impl/VertxContext.java

```diff
@@ -48,9 +50,12 @@ public <T> void put(Key<T> key, T instance) {
 	public <T> T get(Key<T> key) {
 		final io.vertx.core.Context context = Vertx.currentContext();
 		if ( context != null ) {
-			return context.getLocal( key );
+			T local = context.getLocal( key );
+			LOG.tracef( "Getting value %2$s from context for key %1$s", key, local  );
+			return local;
 		}
 		else {
+			LOG.tracef( "Getting null from context for key %s", key );
```

Comment:
This message is a bit confusing as it states we "got" null, but it's the context that's null.
What about "Context get operation skipped for key %s as there is no current context"